src/plugins/fts-squat/fts-backend-squat.c

```diff
@@ -137,6 +147,9 @@ fts_backend_squat_set_box(struct squat_fts_backend *backend,
 	if (backend->full_len != 0)
 		squat_trie_set_full_len(backend->trie, backend->full_len);
 	backend->box = box;
+	ret = squat_trie_open(backend->trie);
+	if (ret < 0)
+		return -1;
```

Comment:
I am puzzled, are we looking at the same code? Please click maybe on "View full changes". If ret >=0, the next line will execute, and that is:

`return squat_trie_get_last_uid(backend->trie, last_uid_r);`